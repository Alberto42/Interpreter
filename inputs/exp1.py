x = 1 + 2
y = 2 * 2
z = 5 - 3
a = x + y