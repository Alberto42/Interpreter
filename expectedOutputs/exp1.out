Right (VInt 0,fromList [(Ident "a",VInt 9),(Ident "v",VInt 1),(Ident "x",VInt 3),(Ident "y",VInt 4),(Ident "z",VInt 2)])
